src/pages/resources/index.mdx

```diff
@@ -85,31 +85,14 @@ Everything you need to learn about, work with, and contribute to IBM.com Library
 ## Design resources
 
 ### IBM.com Library
-
-Request access to the IBM.com Library design kit via [Github repo](https://github.com/carbon-design-system/ibm-dotcom-library-design-kit/issues/new?assignees=ljcarot%2C+wonilsuhibm%2C+oliviaflory&labels=adopter+support&template=request-access.md&title=Design+kit+access+request).
-
 <Row className="resource-card-group">
-  
-  <Column colMd={4} colLg={4} noGutterSm>
-<ResourceCard
-  subTitle="IBM.com Library (components)"
-  actionIcon="launch"             
-  href="https://ibm.box.com/s/fdoqr1qb4agyoys5j8wo19eab60m4ek1"
->
-
-![Sketch icon](../../images/sketch-symbol.svg)
-      
-</ResourceCard>
-</Column>
   <Column colMd={4} colLg={4} noGutterSm>
 <ResourceCard
-  subTitle="IBM.com Shell template"
-  actionIcon="launch"             
-  href="https://ibm.box.com/s/2wk21mi1ia2qbpy9ijb07jn10vgmzcwc"
+  subTitle="Request access to the IBM.com Library (components)"
```

Comment:
```suggestion
  subTitle="Request access to the IBM.com Library design kit"
```

@JennySanchez I removed "(components)" and added "design kit" because it will eventually have multiple themes in there and the shell template.

Reply:
We are good.